Firebase/Messaging/FIRMMessageCode.h

```diff
@@ -27,8 +27,8 @@ typedef NS_ENUM(NSInteger, FIRMessagingMessageCode) {
   kFIRMessagingMessageCodeMessaging003 = 2003,  // I-FCM002003
   kFIRMessagingMessageCodeMessaging004 = 2004,  // I-FCM002004
   kFIRMessagingMessageCodeMessaging005 = 2005,  // I-FCM002005
-  kFIRMessagingMessageCodeMessaging006 = 2006,  // I-FCM002006 - no longer used
```

Comment:
@chliangGoogle  Was this used before for the exact same thing? Or are we okay with re-using message codes?

Reply:
Should be ok. This error code list just describes which number has been used, which can be replaced by a script to do so. Eventually we also want to migrate them to string instead of number so it is easier for developers to see.

Reply:
I wasn't sure what to put for the log warning. @chliangGoogle suggested I just grab some unused codes from that file, and remove the note that they're not used, so that's what I did.